### DISCUSSION TITLE
UDP Send

### DISCUSSION THREAD

**Discussion by syedadil19** [pschatzmann/arduino-audio-tools#1090] [Q&A] [answered]
Hi Phil, I was testing out the communication-udp-send.ino example earlier. But, I can't seem to test it out to hear the audio that is being transmitted. I downloaded the VLC for Android app to try out the streams by writing my network's IP address (udp://@0.0.0.0:7000) but I kept getting "The location udp://@0.0.0.0:7000 cannot be played" or "Multiple media can't be played' prompt.
https://github.com/pschatzmann/arduino-audio-tools/blob/bdc3713fd30878b196e381d1b462c35ecd77b768/examples/examples-communication/udp/communication-udp-send/communication-udp-send.ino
I am connecting the ESP32 and my Android phone to my other phone's hotspot and I could verify the ESP32 was connected to the hotspot by checking the number of devices connected to it. Any clues from your side on how to test this script out?

**Comment by pschatzmann** [comment, marked as answer]
I dont know vlc good enough to be of any help, but your address does not look right to me

You can test the receive with the corresponding arduino sketch or on the desktop e.g with ffmpeg

**Reply by syedadil19** [reply]
That was just an example I put to write here. The actual address I got from ESP32 was 172.20.10.2

Alright, I’ll give the other options a go. Thank you for your reply and I wish you a quick recovery.

**Reply by pschatzmann** [reply]
Yes, but that's equally wrong: I don't expect to see an address, but just a port. The target address is specified in the sending sketch

**Comment by joba-1** [comment]
for me the @ looks strange. It is often used to separate user:password from ip:port. But if you don’t need a user and pw, it is worth trying without the @. Good luck
